ddl/ddl_api.go

```diff
@@ -7564,6 +7584,120 @@ func checkIgnorePlacementDDL(ctx sessionctx.Context) bool {
 	return false
 }
 
+// CreateResourceGroup implements the DDL interface, creates a resource group.
+func (d *ddl) CreateResourceGroup(ctx sessionctx.Context, stmt *ast.CreateResourceGroupStmt) (err error) {
```

Comment:
Please make sure proper privilege are checked for resource group admin. @tiancaiamao, which similar privilege do you think to follow?

Reply:
Super? or use the SEM?

Reply:
We can merge this pr and add the privilege check and other polishing change via other prs.